app/models/manageiq/providers/openstack/storage_manager/cinder_manager.rb

```diff
@@ -58,11 +58,11 @@ def self.ems_type
   end
 
   def self.description
-    @description ||= "Cinder ".freeze
+    @description ||= "Openstack Cinder ".freeze
```

Comment:
The CloudManager and NetworkManager both use `OpenStack` can you match that for all of these?